lib/test.js

```diff
@@ -230,33 +251,23 @@ Test.prototype._publicApi = function () {
 	api.plan = this.plan.bind(this);
 
 	function skipFn() {
-		self._assert();
+		self._assert(Promise.resolve());
```

Comment:
`self._assert()` should work fine here. `Promise.all` converts array members to promise using `Promise.resolve` anyways.